content/en/account_management/teams.md

```diff
@@ -61,6 +61,7 @@ Datadog supports associating the following resources with team handles:
 - [Resource Catalog][14]
 - [Service Catalog][5]
 - [Service Level Objectives][6]
+- Synthetic Tests, Global Variables, Private Locations
```

Comment:
Just flagging this for visibility -- other items in this list are linked to associated documentation pages; is it intentional to leave this new entry un-linked to anything?